bin/bst.ts

```diff
@@ -27,7 +27,14 @@ program
 Global.initializeCLI().then(
     () => program.parse(process.argv)
 ).catch((error) => {
-    LoggingHelper.error(Logger, error);
+    // Request to create pipe or source failed
+    if (error.code === "ETIMEDOUT") {
```

Comment:
Okay, one last tweak I think it should read: "Could not establish connection. Please check your network connection and try again."